2021/d01/p1.py

```diff
@@ -0,0 +1,33 @@
+from pathlib import Path
+from typing import Iterator
+
+INPUT_FILE_PATH = Path('..', 'inputs', '1.txt')
+
+
+def get_measurements_from_input(input_text: str) -> list:
+    """
+    :param input_text: input test to process
+    :return: split measurements from the input
+    """
+    return list(map(int, input_text.splitlines()))
+
+
+def get_depth_measurements_increases(measurements: Iterator[int]) -> int:
+    """
+    :param measurements: measurements to parse
+    :return: count of the number of times a depth measurement increases
+    """
+    couples = zip(measurements[:-1], measurements[1:])
+    return sum(couple[1] > couple[0] for couple in couples)
+
+
+def main():
```

Comment:
Write the I/O logic once, and for each problem implement only the pure logic: A function receiving the input as a string and returning the output as a string.

Then you can maybe think about an automation for running all the problems and uploading the results.

Reply:
And the main function and defining the file's locations and printing the result...